queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
```

Comment:
- [ ] This query is not part of the requirements, Please make sure to remove this.